api/pkg/cli/hub/get_resource.go

```diff
@@ -86,10 +92,26 @@ type ResourceWithVersionData = rclient.ResourceVersionDataResponseBody
 
 type resourceYaml = rclient.ByCatalogKindNameVersionYamlResponseBody
 
+type ArtifactHubPkgResponse struct {
+	Name              string               `json:"name,omitempty"`
+	Data              ArtifactHubPkgData   `json:"data,omitempty"`
+	AvailableVersions []ArtifactHubVersion `json:"available_versions,omitempty"`
+}
+
+type ArtifactHubPkgData struct {
+	PipelineMinVer string   `json:"pipelines.minVersion"`
+	ManifestRaw    string   `json:"manifestRaw"`
+	Platforms      []string `json:"platforms"`
+}
+
+type ArtifactHubVersion struct {
+	Version string `json:"version"`
+	TS      int64  `json:"ts"`
```

Comment:
What does this variable define ?

Reply:
`TS` is defined in Artifact Hub data model but we don't use it in our codebase, so removing it.